### ISSUE TITLE
Since Grabber also downloads video, it should use 'files' instead of 'images'

### ISSUE THREAD

**Issue by adem4ik** [Bionus/imgbrd-grabber#2735]
**Is your feature request related to a problem? Please describe**

There is a lot of places where Grabber speaks about `images`, but videos aren't images.

**Describe the solution you'd like**

`Images` can be renamed to `files`.

**Describe alternatives you've considered**

Also `media files` may work.

**Additional context**

![image](https://user-images.githubusercontent.com/4707112/176389384-b655c0f5-baeb-4097-b9c2-e6b743eded0a.png)
![image](https://user-images.githubusercontent.com/4707112/176389699-de14a4f3-779d-4e91-be18-ef9776a9269a.png)
![image](https://user-images.githubusercontent.com/4707112/176389935-9d02c2a3-be18-46d9-b387-c451612ac772.png)
![image](https://user-images.githubusercontent.com/4707112/176390296-4231218d-17c2-4ce0-aeb4-e1ba8e7f286e.png)
![image](https://user-images.githubusercontent.com/4707112/176390443-5b8c15ef-35f4-40b0-9b33-ac983aceefeb.png)

**Comment by Bionus** [comment]
Makes sense. I like the term "media" personally, but it might not work as well as "file" depending on the context.

**Comment by adem4ik** [comment]
Hm, I don't find `files per page / file limit` weird to be honest. Some more ideas: `item / result / object / content / media item / media data`. I think `item` is best alternative.
